### PR TITLE
Add Home-Assistant app description

### PR DESCRIPTION
**Pull request by Bouni**
Description for another app that serves SpaceAPI json

### CODE REVIEW

**Comment by Bouni** [comment]
Sure, I'll add a screenshot later this evening or tomorrow!

**Comment by Bouni** [comment]
Sorry for the delay, totally forgot about this.
There is no meaningful screenshot as Home-Assistant just serves the SpaceAPI JSON

File: content/apps/homeassistant/contents.lr [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+description: This Home-Assistant integration allows you to serve the SpaceAPI json file from your Home-Assistant instance.
+---
+category: integration
+---
+source_label: GitHub
+---
+source_url: https://github.com/home-assistant/home-assistant/tree/dev/homeassistant/components/spaceapi
+---
+download1_label: Home-Assitant.io
```
Comment (dbrgn) [inline content/apps/homeassistant/contents.lr]: This is probably a typo:

```suggestion
download1_label: Home-Assistant.io
```

File: content/apps/homeassistant/contents.lr [hunk 2]
```diff
@@ -0,0 +1,18 @@
+_model: app
+---
+title: Home-Assistant Integration
+---
+author: Fabian Affolter
```
Comment (dbrgn) [inline content/apps/homeassistant/contents.lr]: Are you affiliated with a hackerspace? Then you could add a `space: <yourspace>` entry :slightly_smiling_face:

File: content/apps/homeassistant/contents.lr [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+---
+description: This Home-Assistant integration allows you to serve the SpaceAPI json file from your Home-Assistant instance.
```
Comment (dbrgn) [inline content/apps/homeassistant/contents.lr]:
```suggestion
description: This Home-Assistant integration allows you to serve the SpaceAPI JSON file from your Home-Assistant instance.
```

File: content/apps/homeassistant/contents.lr (hunk 1 above)
Comment (Bouni) [reply content/apps/homeassistant/contents.lr]: Yes, it is!

**Comment by Bouni** [comment]
💩 I just realized that you merge author and space into one line like "By Fabian Affolter from Reaktor23."

I'm not the author of the Home-Assistant integration (I just fixed some issues with it), but member of the Reaktor23 Hackerspace.

What's your desired way of handling this? Remove the space and space_url?

**Comment by dbrgn** [comment]
Ahh! The main idea is to put the author of the integration into those fields (if there is a main author). Who is that?

If necessary, we can also adjust the model fields to become more flexible.

**Comment by Bouni** [comment]
Fabian Affolter is the original author, I'm not aware if he's member of a hackerspace

A more flexible model would be nice, maybe something like a list of authors and their hackerspaces:

Authors:
Fabian Affolter
Bouni from Reaktor 23 Hackerspace

**Comment by dbrgn** [comment]
Unfortunately I don't think Lektor supports that kind of flexibility for input fields. Would be nice though.

In this case, I think you can simply remove the Reaktor 23 reference, and then we update the app views so that the space is only shown when a value is actually available.

**Comment by rnestler** [comment]
Maybe we should just make the author field it a free form string?

**Comment by dbrgn** [comment]
It already is :slightly_smiling_face:

**Pull request merged**